### DISCUSSION TITLE
How to select all?

### DISCUSSION THREAD

**Discussion by avidseeker** [xournalpp/xournalpp#4353] [Q&A]
selecting all is useful for clearing canvas or shifting whole drawing down by a space. Why ctrl+a doesn't work?

**Comment by netlimpopo** [comment]
Hello!
If you use the nightly build, then this option should already work see #4024. 
Elsewise try Ctrl + a + del.

**Reply by avidseeker** [reply]
ctrl + a + del doesn't work.
using version 1.1.1.

**Reply by netlimpopo** [reply]
You can try version 1.1.2 which was realsed 2 weeks ago. Although I think this function is actually only available in the latest automated nightly build version which can be found at https://github.com/xournalpp/xournalpp/releases/tag/nightly.
